test/e2e/parallel/space_test.go

```diff
@@ -250,19 +253,27 @@ func TestPromoteSpace(t *testing.T) {
 	// make sure everything is ready before running the actual tests
 	awaitilities := WaitForDeployments(t)
 	hostAwait := awaitilities.Host()
-	memberAwait := awaitilities.Member1()
 
 	// when
-	space, _, _ := CreateSpace(t, awaitilities, testspace.WithTierName("base"), testspace.WithSpecTargetCluster(memberAwait.ClusterName))
-	// then
-	VerifyResourcesProvisionedForSpace(t, awaitilities, space.Name, UntilSpaceHasStatusTargetCluster(memberAwait.ClusterName))
+	username := uuid.Must(uuid.NewV4()).String()
+	signup, _ := testsupport.NewSignupRequest(awaitilities).
+		Username(username).
+		Email(username + "@acme.com").
+		ManuallyApprove().
+		RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
+		WaitForMUR().
+		Execute(t).
+		Resources()
+
+	spaceName := signup.Status.CompliantUsername
```

Comment:
I think you could also use the mur name as space name, this is what the [usersignup_controller uses in the creation phase](https://github.com/codeready-toolchain/host-operator/blob/7d449f8bdf5449a58b39bece32c8a00ab6192659/controllers/usersignup/usersignup_controller.go#L706). 

Also should we keep the space provisioned verification?  
```suggestion
	signup, mur := testsupport.NewSignupRequest(awaitilities).
		Username(username).
		Email(username + "@acme.com").
		ManuallyApprove().
		RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
		WaitForMUR().
		Execute(t).
		Resources()
        spaceName := mur.Name 
	VerifyResourcesProvisionedForSpace(t, awaitilities, spaceName)
```

Reply:
Thank you so much, @mfrancisc! 🚀 I will change to use the mur name as space name.

Regarding space provisioned verification, since we are waiting for mur, there will not be space provisioned verification in the Execute function. Unless we change to [ensure mur](https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/signup_request.go#L291). WDYT?

Reply:
Thank you so much, @mfrancisc! 🚀 I will change to use the mur name as space name.

Regarding space provisioned verification, WDTY about using `EnsureMUR` instead of `WaitForMur`? This way, we will have the space related resources verification.

Reply:
> Regarding space provisioned verification, WDTY about using EnsureMUR instead of WaitForMur?

Good point! Indeed it makes sense to me.

Reply:
there are a couple of differences I'm noticing between [VerifyResourcesProvisionedForSpace](https://github.com/codeready-toolchain/toolchain-e2e/blob/716ec177235b9b043eddcb967567f2806e8463a2/testsupport/space/space.go#L115) ( which was used before ) and [VerifySpaceRelatedResources](https://github.com/codeready-toolchain/toolchain-e2e/blob/716ec177235b9b043eddcb967567f2806e8463a2/testsupport/user_assertions.go#L222) which will be executed by `EnsureMUR` . 

`VerifySpaceRelatedResources` only [verifies the Space.Spec.targetCluster field](https://github.com/codeready-toolchain/toolchain-e2e/blob/716ec177235b9b043eddcb967567f2806e8463a2/testsupport/user_assertions.go#L241)

`VerifyResourcesProvisionedForSpace` checks for a [specific target cluster in the status of the space](https://github.com/codeready-toolchain/toolchain-e2e/blob/716ec177235b9b043eddcb967567f2806e8463a2/testsupport/space/space.go#L146)  , and also [waits for the provisioned namespaces list to be populated](https://github.com/codeready-toolchain/toolchain-e2e/blob/716ec177235b9b043eddcb967567f2806e8463a2/testsupport/space/space.go#L174) , which doesn't seem to be checked by the other function. Thus there might be some side effects if we replace those checks , but I'm not 100% sure.